src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ContinuousVariableCard.js

```diff
@@ -407,12 +407,22 @@ export default compose(
       setId,
       totalDocs,
       variable: {
+        active_chart,
         bins = {},
         continuousBinType,
         customSurvivalPlots,
         isSurvivalCustom,
       },
     }) => {
+      // prevent survival API requests on mount
+      // when a different plot is selected
+      if (active_chart !== 'survival') {
+        return {
+          survivalPlotValues: [],
+          survivalTableValues: [],
+        }
+      };
```

Comment:
semicolon should be after the return - linter issue?

Reply:
yikes, good eye